modules/lymphgen/1.0/src/curate_svs_for_lymphgen.R

```diff
@@ -1,32 +1,51 @@
 source("/projects/rmorin/projects/gambl-repos/gambl-lhilton/src/R/GAMBLR_libs.R")
 library(GAMBLR)
 
+# different cases for inputs
+#   1) fish w/ sample_id + svar master -- has_fish & has_sv
+#   2) fish w/ sample_id + manta -- has_fish & has_sv
+#   3) fish w/o sample_id (like empty fish) + svar master -- ~has_fish & has_sv
+#   4) fish w/o sample_id (like empty fish) + manta -- ~has_fish & has_sv
+#   5) fish w/ sample_id + empty sv -- has_fish & ~has_sv
+
 all_fish <- snakemake@input[["fish"]]
-one_svar_master <- snakemake@input[["svar_master"]]
+sv <- snakemake@input[["sv"]]
+
+if (str_detect(sv, "manta")) { # manta
+  sv <- fread(sv, skip = 130) %>% 
```

Comment:
A more reliable way to do this (in case manta ever produces a different number of header lines) is to use `read_tsv(..., comment = "#", col_names = c("CHROM_A", "START_A", ...))`. You have to specify the col_names because the column header line in the bedpe has a "#". (Unless they've updated the behaviour of read_tsv to accept `##` as a comment character, in which case you would just need to switch fread to read_tsv. Test it out.)

Reply:
I kept it as fread with `skip = "CHROM"` cause I think it's quicker than read_tsv but if read_tsv is still better in this situation then I can change it :)